src/components/Header/Polling.tsx

```diff
@@ -43,6 +57,17 @@ const StyledPollingDot = styled.div<{ warning: boolean }>`
   transition: 250ms ease background-color;
 `
 
+const StyledGasDot = styled.div`
+  width: 4px;
```

Comment:
we're not super good about this but we usually try to keep the properties sorted

Reply:
Do we have a guideline for the ordering?

Reply:
just alphabetical